demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -298,26 +301,46 @@ export class DemoMeetingApp
 
   meetingLogger: Logger | undefined = undefined;
 
-  constructor() {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    (global as any).app = this;
+  // If you want to make this a repeatable SPA, change this to 'spa'
```

Comment:
Could we add more documentation here? Such as if you do Shift + Leave Meeting it will trigger halt to clean up the demo app?